rebasehelper/build_tools/koji_tool.py

```diff
@@ -90,8 +90,13 @@ def _scratch_build(cls, source, **kwargs):
         task_dict = cls.koji_helper.watch_koji_tasks(session, [task_id])
         task_list = []
         package_failed = False
+        build_error = ''
         for key in six.iterkeys(task_dict):
             if task_dict[key] == koji.TASK_STATES['FAILED']:
+                try:
+                    info = session.getTaskResult(key)
```

Comment:
There is no reason to assign result of `getTaskResult()` method to a variable, it's not used anyway.

Reply:
You are right, but I consider it better for readability, to be more obvious, that it actually gets the results. I can get rid of the assignment.

Reply:
If you think it would be confusing, add a comment with explanation.